tests/ctst/world/Zenko.ts

```diff
@@ -147,18 +472,242 @@ export default class Zenko extends World {
         };
     }
 
-    /**
-     * Utility function to prepare Zenko
-     * @param {object} parameters - the client-provided parameters
-     * @returns {undefined}
-     */
-    static async init(parameters: any) {}
-
     /**
      * Cleanup function for the Zenko world
      * @returns {undefined}
      */
-    static async teardown() {}
+    static async teardown() { }
+
+    async metadataSearchResponseCode(userCredentials : any, bucketName : string) {
+        return await this.awsS3GetRequest(
+            `/${bucketName}/?search=${encodeURIComponent('key LIKE "file"')}`,
+            userCredentials,
+        );
+    }
+
+    async awsS3GetRequest(path: string, userCredentials: any) {
+        const credentials : any = {
+            accessKeyId: userCredentials.AccessKeyId,
+            secretAccessKey: userCredentials.SecretAccessKey,
+        };
+        if (userCredentials.SessionToken) {
+            credentials['sessionToken'] = userCredentials.SessionToken;
+        }
+        const interceptor = aws4Interceptor({
+            region: 'us-east-1',
+            service: 's3'
+        }, credentials);
+
+        axios.interceptors.request.use(interceptor);
+        const protocol = this.parameters.ssl === false ? 'http://' : 'https://';
+        try {
+            const response = await axios.get(`${protocol}s3.${this.parameters.subdomain || Constants.DEFAULT_SUBDOMAIN}` + path);
+            console.log(response.data);
+            return { statusCode: response.status, data: response.data }
+        } catch (err: any) {
+            console.log(err.response);
+            return { statusCode: err.response.status, data: err.response.data };
+        }
+    }
+
+    async restoreObjectResponseCode() {
+        this.addCommandParameter({ restoreRequest: "Days=1" });
+        return await S3.restoreObject(this.getCommandParameters());
+    }
```

Comment:
This would go into another utils file IMHO, this is not really related to the Zenko namespace, but more a convenient way to easily call all APIs with the right parameters using the same step definition.